js/everything.js

```diff
@@ -474,7 +473,13 @@ $(function() {
             );
         }
     }
-
+    if ( /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) )
+    {
+        $("#container").ready(function() {
+          $("#container").html("<p></p>");
```

Comment:
We shouldn't add a paragraph inside of a header. You can add a `<p>` tag to the HTML that's hidden by default, then here do `$("change-mobile").show();`